src/app.js

```diff
@@ -39,6 +40,85 @@ module.exports = function ({ db, config, logger }) {
 
     res.send(`200 TLS OK\r\n${str}\r\n`)
   }))
+  app.get('/tls-json', wrap(async (req, res) => {
```

Comment:
Is there a reason to call this `/tls-json` rather than `/templates`?
It would be nice to use a real and clearer name rather than amcp names. 
This doesn't provide the `200 TLS OK\r\n` prefix that is produced for the `/tls` endpoint, so it isnt a dropin replacement as is, and may as well ditch the amcp naming and use something friendlier

Reply:
I'm all for naming it /templates!